estate/models/estate_property.py

```diff
@@ -0,0 +1,30 @@
+from odoo import models, fields
+from datetime import date
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(string="Title", required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, readonly=True, default=date.today() + relativedelta(months=3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer(string='Living Area (m2)')
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer(string='Garden Area (m2)')
+    garden_orientation = fields.Selection(
+        string='garden_orientation',
+        selection=[('north', 'North'), ('south', 'South'),
+        ('east', 'East'), ('west', 'West')])
+    active = fields.Boolean(default=False)
```

Comment:
```suggestion
    active = fields.Boolean()
```
False is already the default for Boolean